### ISSUE TITLE
Why can't I use ctrl+alt+A as a hotkey?

### ISSUE THREAD

**Issue by mzso** [Belphemur/SoundSwitch#423]
Hi!

It seems like that I can use literally every other key with ctrl+alt. A tad annoying.

**Comment by Belphemur** [comment]
Hello,

Surely because it's already registered by another app.

**Comment by mzso** [comment]
[**@Belphemur**](https://github.com/Belphemur) commented on [2020. ápr. 24. 20:36 CEST](https://github.com/Belphemur/SoundSwitch/issues/423#issuecomment-619178836 "2020-04-24T18:36:40Z - Replied by Github Reply Comments"):
> Hello,
> 
> Surely because it's already registered by another app.

Doesn't seem like it. I havent used it for anything and nothing happens when I press it.
Though I can't use it in Firefox either, so something's up.

**Comment by Belphemur** [comment]
Yep, sound like one application has register it.

SoundSwitch isn't able to override a previously set shortcut.

**Comment by mzso** [comment]
[**@Belphemur**](https://github.com/Belphemur) commented on [2020. ápr. 27. 22:13 CEST](https://github.com/Belphemur/SoundSwitch/issues/423#issuecomment-620208670 "2020-04-27T20:13:53Z - Replied by Github Reply Comments"):
> Yep, sound like one application has registered it.
> 
> SoundSwitch isn't able to override a previously set shortcut.

It doesn't to me. Nothing uses it. ctrl+shift+alt+a also doesn't work. I can't imagine any app registering that...

Also overriding and more aggressive hotkey usage would be a fine thing. I got accustomed to use win+a, but now after upgrading to Win10 I kept getting a message that it can't be registered.
Also, even if a key is registered apparently an app can override it as long as it's on top.
